geocoder/geonames.py

```diff
@@ -94,11 +95,12 @@ def _build_params(self, location, provider_key, **kwargs):
             'maxRows': kwargs.get('maxRows', 1),
         }
         # check out for bbox in kwargs
-        bbox = kwargs.pop('bbox', None)
+        bbox = kwargs.pop('proximity', None)
```

Comment:
👍 That's way better using `proximity` instead of a specific `bbox` or `bounds` term.